obs-websocket-dotnet/VirtualCamStatus.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace OBSWebsocketDotNet
+{
+    public class VirtualCamStatus
+    {
+        /// <summary>
+        /// The current virtual camera status
+        /// </summary>
+        [JsonProperty(PropertyName = "isVirtualCam")]
+        public bool IsActive { get; set; }
+
+        /// <summary>
+        /// Time elapsed since virtual camera started (only present if virtual cam currently active)
+        /// </summary>
+        [JsonProperty(PropertyName = "virtualCamTimecode")]
+        public string Timecode { get; set; }
+
+        /// <summary>
+        /// Builds the object from the JSON response body
+        /// </summary>
+        /// <param name="data">JSON response body as a <see cref="JObject"/></param>
+        public VirtualCamStatus(JObject data)
```

Comment:
Please also add a default constructor (With no arguments), otherwise this will fail JSON deserialization.